x-pack/plugins/synthetics/e2e/journeys/data_view_permissions.ts

```diff
@@ -48,10 +48,10 @@ journey('DataViewPermissions', async ({ page, params }) => {
   step('Click explore data button', async () => {
     await page.click(byTestId('uptimeExploreDataButton'));
     await waitForLoadingToFinish({ page });
-    await page.waitForSelector(`text=${permissionError}`, TIMEOUT_60_SEC);
-    expect(await page.$(`text=${permissionError}`)).toBeTruthy();
   });
-});
 
-const permissionError =
-  "Unable to create Data View. You don't have the required permission, please contact your admin.";
+  step('it renders for viewer user as well', async () => {
```

Comment:
Why remove the intent of the original test? Does the new fix Emilio put in handle this use case so that people without permissions can use data views now?